frontends/web/src/routes/account/info/buyCTA.tsx

```diff
@@ -0,0 +1,44 @@
+import { FunctionalComponent, h } from 'preact';
+import { route } from 'preact-router';
+import { Coin } from '../../../api/account';
+import { Button } from '../../../components/forms';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { Balances } from '../summary/accountssummary';
+import * as styles from './buyCTA.module.css';
+
+interface BuyCTAProps {
+    code?: string;
+    unit?: string;
+}
+
+type Props = BuyCTAProps & TranslateProps;
+
+const BuyCTAComponent: FunctionalComponent<Props> = ({code, unit, t}) => {
+    const onCTA = () => route(code ? `/buy/info/${code}` : `/buy/info`);
+    return (
+    <div className={`${styles.main} columns-container`}>
+        <h3 className="column">{t('accountInfo.buyCTA.information.looksEmpty')}</h3>
+        <h3 className="column">{t('accountInfo.buyCTA.information.start')}</h3>
+        <div className="column">
+            <Button primary onClick={onCTA}>{unit ? t('accountInfo.buyCTA.buy', {unit}) : t('accountInfo.buyCTA.buyCrypto')}</Button>
+        </div>
+    </div>);
+};
+
+export const BuyCTA = translate<BuyCTAProps>()(BuyCTAComponent);
+
+const isBitcoinCoin = (coin: Coin) => (coin === 'BTC') || (coin === 'TBTC');
```

Comment:
I thought we can use https://github.com/digitalbitbox/bitbox-wallet-app/blob/d6b33a6d25e394e11bcb8055db41ff77129769ca/frontends/web/src/routes/account/utils.ts#L20

but Coin is coinCode but uppercase https://github.com/digitalbitbox/bitbox-wallet-app/blob/d6b33a6d25e394e11bcb8055db41ff77129769ca/frontends/web/src/api/account.ts#L27-L31

not sure what to do, maybe keep this function but move to `routes/accounts/utils.ts`, @x1ddos what do you think?

also `isBitcoinCoin` name is funny but somehow makes sense :)